msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -267,6 +277,11 @@ Provide feedback and report any issues with the Teams Toolkit debugging experien
 
 #### Mobile debugging
 
+> [!IMPORTANT]
+> To debug your app in Outlook for Android:
+> 1. Select the **More** icon in the Teams mobile client and open your sideloaded app to run within the Outlook app. 
+> 1. Follow the steps 7 through 9.
+
```

Comment:
Instead of this "IMPORTANT" callout, could we split this mobile debugging section into two headings, one each for M365 app and Outlook?